http/protocolversioning.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<title>Test HTTP's versioning scheme</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id=log></div>
+<script>
+[
+  "HTTP/0.7 280",
+  "HTTP/1.0 280",
+  "HTTP/1.11 280",
+  "HTTP/1.2 280",
+  "HTTP/1.7 280",
+  "HTTP/2.0 280",
```

Comment:
HTTP/!1 is a wire-incompatible with HTTP/1.x, so all bets are off for this and HTTP/5, etc. These should be errors, as should be plain "http" and "splitonspace".